windows/src/support/kmdevlink/UfrmMain.pas

```diff
@@ -12,7 +12,9 @@ interface
 
 const
   SStatusSiteURL = 'https://status.keyman.com';
-  SSearchURL = 'https://github.com/keymanapp/%s/issues/%s';
+  SOpenIssueURL = 'https://github.com/keymanapp/%s/issues/%d';
+  SOpenAllIssuesURL = 'https://github.com/keymanapp/%s/issues';
+  SIssueSearchURL = 'https://github.com/keymanapp/%s/issues?q=state%%3Aopen%%20%s';
   SAddIssueURL = 'https://github.com/keymanapp/keyman/issues/new';
```

Comment:
Should this follow the pattern for consistency?
```suggestion
  SAddIssueURL = 'https://github.com/keymanapp/%s/issues/new';
```

Reply:
Nah, right now there's no way to select repo. In the future maybe. This app has lower standards of code because it's internal-only and I really don't want to invest too much into it :grin: